src/style.css

```
@@ -0,0 +1,118 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+  color: #000;
+}
+
+.product {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 30px 14px 14px;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.product__image {
+  height: 134px;
+  width: 160px;
+  margin-bottom: 45px;
+}
+
+.product__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 15px;
+}
+
+.product__code {
+  align-self: start;
+  font-size: 11px;
+  line-height: 13px;
+  margin-top: 5px;
+}
+
+.product__rate {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  margin-top: 15px;
+}
+
+.product__rate_stars {
+  display: flex;
+}
+
+.stars__star {
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: center;
```

you can write this in one line
```suggestion
  background: url(images/star.svg) no-repeat center;
```